bucket/latex.json

```diff
@@ -1,22 +1,33 @@
 {
     "homepage": "https://miktex.org",
-    "version": "2.9.6942",
+    "version": "2.9.7100",
     "license": {
         "identifier": "Freeware",
         "url": "https://miktex.org/copying"
     },
-    "url": "https://ftp.fau.de/ctan/systems/win32/miktex/setup/windows-x86/miktex-portable-2.9.6942.exe#/dl.7z",
-    "hash": "20e42d52e7601f7b0abaf042408a18b39ffd84b3bb5c2a51f9b70a6555ea5897",
+    "url": "https://ftp.fau.de/ctan/systems/win32/miktex/setup/windows-x86/basic-miktex-2.9.7100.exe",
+    "hash": "893505ea259a77a8a3744b64a1975a20abc57e4d270edc2d03affad36b3427dd",
+    "installer": {
+        "args": "--portable=\"$dir\" --unattended --private"
+    },
+    "bin": [
+        [
+            "texmfs\\install\\miktex\\bin\\miktex-console.exe",
+            "miktex",
```

Comment:
It is miktex-console, not miktex.

Add miktex-console without renaming and then new with renaming

Reply:
It is created by MiKTeX with name 'miktex-portable', so I rename it to just 'miktex'. 'miktex-console' itself can be accessed since it's in `PATH`.

Reply:
Yeah but without parameters

Reply:
So for this one with parameters, 'miktex' is preferred, and for general usage, 'miktex-console' can be used.

Reply:
I do not agree, but whatever.